### ISSUE TITLE
Clipboard sharing conflicts with Mac Universal Clipboard

### ISSUE THREAD

**Issue by allanjackson** [symless/synergy#5820]
### Operating Systems ###
Server: Mac Sierra
Client: Mac Sierra

### Synergy Version ###
1.8.7

### General Info ###
When I'm using 2 Macs with Universal clipboard/handoff enabled, Synergy's clipboard sharing feature locks up. I'm guessing that having 2 programs simultaneously watching and modifying the clipboard is causing an infinite loop or something.

To reproduce:
1. Enable clipboard sharing on Macs and in Synergy.
2. Copy on one Mac.
3. Paste on the other Mac.
4. Spinning Beach Ball of Death

### Other info ###
I've been having this problem ever since Sierra came out. I can turn off Synergy's clipboard sharing to temporarily fix the issue, but I also frequently connect to a Windows computer, so I need the feature on for that.

**Comment by nlyan** [comment]
We introduced a checkbox on the server configuration dialog (under Advanced Server Settings) where you can disable clipboard sharing as a temporary workaround to this issue.

**Comment by allanjackson** [comment]
Yeah, that's the workaround I use. Unfortunately I also connect to a Windows computer, so I can't just leave the feature disabled.
